task_034.py

```diff
@@ -0,0 +1,29 @@
+# Задача 34:  Винни-Пух попросил Вас посмотреть, есть ли в его стихах ритм. Поскольку разобраться в его кричалках не настолько просто, насколько легко он их придумывает,
+#  Вам стоит написать программу. Винни-Пух считает, что ритм есть, если число слогов (т.е. число гласных букв) в каждой фразе стихотворения одинаковое. 
+# Фраза может состоять из одного слова, если во фразе несколько слов, то они разделяются дефисами. Фразы отделяются друг от друга пробелами. 
+# Стихотворение  Винни-Пух вбивает в программу с клавиатуры. В ответе напишите “Парам пам-пам”, если с ритмом все в порядке и “Пам парам”, если с ритмом все не в порядке
+
+# *Пример:*
+
+# **Ввод:** пара-ра-рам рам-пам-папам па-ра-па-да    
+#     **Вывод:** Парам пам-пам  
+
+def is_rhyme(phrase):
+    list_v =[]
+    List_phrase = phrase.split()
+    for i in List_phrase:
+        s = 0
+        k = [s+1 for letter in i if letter in "уеыаоэяию"]
```

Comment:
Этот ваш кот не будет увеличивать переменную `s`, он просто будет записывать единицы в список `k`, а значит можно сделать проще, учесть при этом, что  истина эквивалентная единице.
```suggestion
        k = [letter in "уеыаоэяию" for letter in i]
```
При этом дальнейший код менять не нужно, а функциональность не изменится.